setup.py

```diff
@@ -1,10 +1,10 @@
 from setuptools import setup
 
-requirements = ["dataclasses==0.8;python_version<'3.7'"]
+requirements = ["dataclasses==0.8,~=0.6;python_version<'3.7'"]
```

Comment:
Why pin to a specific version at all? Our use of `dataclasses` is pretty minimal, and should realistically work with any version.


Why not just:
```python
requirements = ["dataclasses;python_version<'3.7'"]
```

Could also just do:
```python
requirements = ["dataclasses>=0.6;python_version<'3.7'"]
```